src/main/resources/credit_advisory_schema2.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TYPE user_role AS ENUM ('advisor', 'applicant');
+CREATE TYPE status AS ENUM ('new', 'assigned', 'on_hold', 'approved', 'canceled', 'declined');
+CREATE TYPE role AS ENUM ('associate', 'partner', 'senior');
+
+
+create table users
+(
+    id              bigint,
+    user_role       user_role,
+    first_name      varchar(255),
+    last_name       varchar(255),
+    ssn             bigint,
+    phone_number_id bigint,
+    email varchar(255),
+    CONSTRAINT PK_applicant PRIMARY KEY (id),
+    CONSTRAINT UQ_sales_group_name UNIQUE (first_name),
+    CONSTRAINT UQ_sales_group_name UNIQUE (email)
+);
+
+create table address
```

Comment:
If you use `s` at the end of the table names. Let's keep this style for this table